### PR TITLE
Updated Meta Tags & SEO Optimizations for faces and platforms

### PR DESCRIPTION
**Pull request by linkfang**
For https://github.com/jakartaee/jakarta.ee/issues/1478

### CODE REVIEW

**Comment by linkfang** [comment]
Got +1 from Kristy, https://github.com/jakartaee/jakarta.ee/issues/1478#issuecomment-1142628856

File: faces/_index.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
 title: "Jakarta Faces"
-summary: "Jakarta Faces defines an MVC framework for building user interfaces for web applications, 
-including UI components, state management, event handing, input validation, page navigation, and 
+summary: "Jakarta Faces (JSF) defines an MVC framework for building user interfaces for web applications,
```
Comment (ivargrimstad) [inline faces/_index.md:3]: Remove "(JSF)"

File: platform/10/_index.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ---
 title: "Jakarta EE Platform 10 (under development)"
 date: 2021-08-11
-summary: "Release of the Jakarta EE 10 Platform"
+summary: "Jakarta EE Platform 10 defines a standard platform for hosting Jakarta EE (Java EE) applications. Find specifications and compatible implementations."
```
Comment (ivargrimstad) [inline platform/10/_index.md:4]: Remove (Java EE)

File: platform/8/_index.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ---
 title: "Jakarta EE Platform 8"
 date: 2019-09-10
-summary: "Initial release of the Jakarta EE 8 Platform"
+summary: "Jakarta EE Platform 8 defines a standard platform for hosting Jakarta EE (Java EE) applications. Find specifications and compatible implementations."
```
Comment (ivargrimstad) [inline platform/8/_index.md:4]: Remove (Java EE)

File: platform/9.1/_index.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
 ---
 title: "Jakarta EE Platform 9.1"
 date: 2021-05-14
-summary: "Release of the Jakarta EE 9.1 Platform"
+summary: "Jakarta EE Platform 9.1 defines a standard platform for hosting Jakarta EE (Java EE) applications. Find specifications and compatible implementations."
```
Comment (ivargrimstad) [inline platform/9.1/_index.md:4]: Remove (Java EE)

File: platform/9/_index.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
 ---
 title: "Jakarta EE Platform 9"
 date: 2020-11-20
-summary: "Release of the Jakarta EE 9 Platform"
+summary: "Jakarta EE Platform 9 defines a standard platform for hosting Jakarta EE (Java EE) applications. Find specifications and compatible implementations."
```
Comment (ivargrimstad) [inline platform/9/_index.md:4]: Remove (Java EE)

**Review by ivargrimstad — CHANGES_REQUESTED** [review]

**Review by ivargrimstad — APPROVED** [review]

**Pull request merged**